vscode/src/toitExec.ts

```diff
@@ -36,12 +40,20 @@ async function executeRunCommand(ctx: Context, device?: Device) {
 
   if (!device) return;  // Device selection prompt dismissed.
 
-  try {
-    ctx.output.startDeviceOutput(device);
-    toitSpawn(ctx, "dev", "-d", device.name, "run", filePath);
+  try {    
+    const fileName = basename(filePath);
```

Comment:
Move this below after the `toitExecFile`. This way it's clearer that it's not used for the exec.